framework/src/play/PropertiesConfLoader.java

```diff
@@ -7,10 +7,7 @@
 import play.utils.OrderSafeProperties;
 import play.vfs.VirtualFile;
 
-import java.util.HashSet;
-import java.util.Map;
-import java.util.Properties;
-import java.util.Set;
+import java.util.*;
```

Comment:
My IntelliJ Idea changed this due to `Objects.requireNonNull()`. Should I adjust my IDE settings?

Reply:
There are 20 occurences of the abbreviated `java.util.*` so it seems to be the way in RePlay.

Reply:
> so it seems to be the way in RePlay.

I don't think so. It's just an IDE settings to migrate to use `import something.*` instead of too much `import something.foo; import something.bar; ...` statements.

I'll change this to explicit import.

Reply:
And here is it:
* `Class count to use import with '*'` and
* `Names count to use static import with '*'`
 https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports

Reply:
Far better now! :+1: